### ISSUE TITLE
Upgrade sweetalert 1.x to sweetalert 2.x

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/codebuddies#1014]
I am filing this bug after discussion in https://github.com/codebuddies/codebuddies/issues/821 (adding a "sign in using Github" link on the sweetalert) surfaced the fact that we need to upgrade to sweetalert 2.x before we can support having both a Github and Slack sign-in actions on the sweetalert.

The task:
- upgrade to sweetalert 2.x from 1.x
- Guidelines to follow: https://sweetalert.js.org/guides/#upgrading-from-1x

**Comment by lpatmo** [comment]
Update: Oh wait, actually it looks like we have https://atmospherejs.com/patrickml/swal installed, which supposedly supports sweetalert 2. 

Wonder if someone can confirm that the following works:
```
swal({
  title: '<strong>HTML <u>example</u></strong>',
  type: 'info',
  html:
    'You can use <b>bold text</b>, ' +
    '<a href="//github.com">links</a> ' +
    'and other HTML tags',
  showCloseButton: true,
  showCancelButton: true,
  focusConfirm: false,
  confirmButtonText:
    '<i class="fa fa-thumbs-up"></i> Great!',
  confirmButtonAriaLabel: 'Thumbs up, great!',
  cancelButtonText:
    '<i class="fa fa-thumbs-down"></i>',
  cancelButtonAriaLabel: 'Thumbs down',
})```

**Comment by eragon512** [comment]
hi, i'd like to take up this issue, if its available

**Comment by eragon512** [comment]
Sweetalert Screenshot with the above markup:

![screenshot_2018-10-14 codebuddies org codebuddies org](https://user-images.githubusercontent.com/9765685/46915732-2e7d2980-cfcd-11e8-9229-37154565a9b9.png)

**Comment by thunatos** [comment]
I would like to work on this issue.

**Comment by thunatos** [comment]
Hello Ipatmo,

I went ahead and confirmed that 

swal({
  title: '<strong>HTML <u>example</u></strong>',
  type: 'info',
  html:
    'You can use <b>bold text</b>, ' +
    '<a href="//github.com">links</a> ' +
    'and other HTML tags',
  showCloseButton: true,
  showCancelButton: true,
  focusConfirm: false,
  confirmButtonText:
    '<i class="fa fa-thumbs-up"></i> Great!',
  confirmButtonAriaLabel: 'Thumbs up, great!',
  cancelButtonText:
    '<i class="fa fa-thumbs-down"></i>',
  cancelButtonAriaLabel: 'Thumbs down',
})

Works in the code base with no errors. 

Looks like sweetAlert 2 is supported already. 👍

**Comment by lpatmo** [comment]
@thunatos Thanks for looking! I tried it out myself; unless I'm missing another setting, it looks like the HTML isn't supported, unfortunately: 

```
![image](https://user-images.githubusercontent.com/4512699/47251248-c96b6e80-d3e6-11e8-9598-7380e36da02e.png)
```
```
+++ b/client/templates/study_groups/study_group_discussion.js
@@ -107,7 +107,24 @@ Template.studyGroupDiscussion.events({
 
         Meteor.call('discussions.remove', data, function(error, result) {
           if (result) {
-            swal("Poof!", "Your Discussion has been successfully deleted!", "success");
+            //swal("Poof!", "Your Discussion has been successfully deleted!", "success");
+            swal({
+  title: '<strong>HTML <u>example</u></strong>',
+  type: 'info',
+  html:
+    'You can use <b>bold text</b>, ' +
+    '<a href="//github.com">links</a> ' +
+    'and other HTML tags',
+  showCloseButton: true,
+  showCancelButton: true,
+  focusConfirm: false,
+  confirmButtonText:
+    '<i class="fa fa-thumbs-up"></i> Great!',
+  confirmButtonAriaLabel: 'Thumbs up, great!',
+  cancelButtonText:
+    '<i class="fa fa-thumbs-down"></i>',
+  cancelButtonAriaLabel: 'Thumbs down',
+})
           } else {
             swal("Oops something went wrong!", error.error + "\n Try again", "error");
           }```

**Comment by thunatos** [comment]
After further research, it appears that there required a new plugin. I went ahead and upgraded to sweetAlert2 with a package wrapper by performing:

--- meteor add cleandersonlobo:sweetalert2

Here are the results:

![image](https://user-images.githubusercontent.com/831191/47251954-e60ca400-d3f0-11e8-8c9d-ceec8f2c9eb0.png)

**Comment by lpatmo** [comment]
@thunatos That looks great! Do you want to create a PR only adding the new packages? swal 2.0 support means we'd finally be able to tackle #821 now.

**Comment by thunatos** [comment]
Hi @lpatmo I have submitted my pull requests yesterday. I tested it out on multiple features and I also had to create a few different cases, say if users click off the overlay, or dismiss the option, or cancel, it will alert users with an info screen instead of just simply "closeOnConfirm". 

I would like to get a few more pull requests in for Hacktoberfest. So let me know and I hope that the changes are acceptable.
